### DISCUSSION TITLE
No thumbnail preview in Finder

### DISCUSSION THREAD

**Discussion by s-zh** [iterate-ch/cyberduck#15454] [Q&A] [answered]
I updated my Mac to Sonoma 14.2, but webdav no longer displays thumbnails in Finder. Has anyone encountered the same problem?

**Comment by websnowtea** [comment]
same issue with 14.2.1. 
Mountain Duck 4.15.1 (21679)

**Reply by DannyAmmon** [reply]
I can confirm this, same OS and software version, thumbnails gone. Finder settings are set to show icon preview, and it does on my hard drive, but not on all of my Mountain Duck drives.

**Reply by DannyAmmon** [reply]
Updated to Mountain Duck 4.15.2 today, problem persists.

**Reply by will-huang9531** [reply]
It seems that Apple has modified its WindowServer strategy in the macOS14.2.1, which may affect the display of thumbnails. 
Other cases also show that Finder seems to have some issues with its thumbnails, and it is unclear whether Apple has done anything about it.
Anyway, my mountainduck has been in an unusable state since this macOS14.2.1 update.

**Comment by dkocher** [comment]
Check your setting in _Finder → View → Show View Options → Show Icon Preview_.

**Reply by websnowtea** [reply]
thanks for getting back! 
Yes, the Show Icon Preview was enabled.

**Reply by will-huang9531** [reply]
same issue，When I  updated my Mac to Sonoma 14.2，and mount a network disk using the webdav, the Icon preview doesn't work in finder, so I cannot see the thumbnails. I'm very confident that “Finder → View → Show View Options → Show Icon Preview” has been enabled. Please try to reproduce and fix this issue.
<img width="1183" alt="iShot_2024-01-05_09 11 01" src="https://github.com/iterate-ch/cyberduck/assets/103096674/686b5e09-1c08-40f1-a83b-da2631732816">

**Comment by dkocher** [comment]
I can confirm that on macOS 14.2 and later previews are no longer loaded in Finder for files on network volumes as is the case for volumes mounted with Mountain Duck.

**Reply by will-huang9531** [reply]
Thanks for your reply!
However, network volumes(webdav) mounted directly by finder can load previews normally, while those volumes mounted with  Mountain Duck cannot load previews. 
Is there any way to make the previews on the network volume mounted with Mountain Duck load normally on macOS 14.2 and later？
<img width="1392" alt="iShot_2024-02-24_14 40 53" src="https://github.com/iterate-ch/cyberduck/assets/103096674/f184df4a-9a41-4390-831d-ff467031216a">

**Comment by websnowtea** [comment]
Issue is still not resolved on Mountain Duck 4.15.4 (21882) with MacOS 14.4.
Are we able to open an issue with this?

**Comment by dkocher** [comment, marked as answer]
As a workaround for this issue please set the custom mount location outside the default `~/Library/Group Containers/…` and instead set it to a folder in your user home or another folder not protected by application group entitlement. The default will change in an upcoming bugfix release to `~/Library/Application Support/Mountain Duck`.

**Reply by liangzx96** [reply]
Can you provide some instructions on how to change the mount location? I don't find a way to change this setting. Thank you

**Reply by liangzx96** [reply]
Never mind... I found it's located at Sync pane of Preferences.

**Comment by liangzx96** [comment]
I can confirm changing mount point solves this issue!
